File: readability/readability.c
#include <ctype.h>
#include <cs50.h>
#include <math.h>
#include <stdio.h>
#include <string.h>

int level(string text);

int main()
{
    string text = get_string("Text: ");

    int letters = calc_letters(text);
    int words = calc_words(text);
    int sentences = calc_sentences(text);

    int grade = 0.0588 * L - 0.296 * S - 15.8
}

File: sentimental-readability/readability.py
from cs50 import get_string

text = get_string('text: ')

letters = sum(1 for char in text if char.isalpha())

words = text.split()
num_words = len(words)

sentences = text.count('.') + text.count('!') + text.count('?')

L = (letters / num_words) * 100

S = (sentences / num_words) * 100

grade = 0.0588 * L - 0.296 * S - 15.8

grade = round(grade)

if grade < 1:
    print('Before Grade 1')
elif grade > 16:
    print('Grade 16+')
else:
    print(f'Grade {grade}')
